ui-grid/ui-grid.d.ts

```diff
@@ -1064,6 +1074,15 @@ declare module uiGrid {
         (scrollEvent: JQueryMouseEventObject): void;
     }
 
+    export interface sortChangedHandler {
+        /**
+         * Sort change event callback
+         * @param {IGridInstance} grid instance
+         * @param {IGridColumn} array of gridColumns that have sorting on them, sorted in priority order
+         */
+        (grid: IGridInstance, columns: IGridColumn[]): void;
```

Comment:
Um, not that I know of... My workflow is to keep my branch around until the PR has been merged or closed. Then I delete the branch. You don't have to delete the repository on GitHub unless you are never again going to work with it again. If you want to work on a different PR, you just create a new branch off of master. 

You can update your master branch by creating a remote to what is usually called upstream which you set up to be what you forked. In this case, borisyankov/DefinitelyTyped. See https://help.github.com/articles/fork-a-repo/, it outlines some of this. With the upstream remote, you can pull that into your master branch whenever you want to create a new branch for a future PR.

Hope this helps. :) Anyway, I think your best bet now is to close this one and create a new PR with the proposed changes.